.dockerignore

```diff
@@ -1,5 +1,5 @@
-*/target*
-*/.vscode*
-*/.git*
-*/.github*
-*/.devcontainer*
\ No newline at end of file
+**/target*
```

Comment:
Why do we need this file in both root and mirrord-agent/?

Reply:
Apparently we don't.